content/pages/operator/community.md

```diff
@@ -0,0 +1,81 @@
+Title: Community
+URL: operator/community.html
+save_as: operator/community.html
+template: operator/community
+
+## Support ##
+
+The Solr Operator community provides user support for free through the [users mailing list](#mailing-lists-chat) and [slack channels](#slack).
+
+## Mailing Lists & Chat ##
+
+We use [mailing lists](https://apache.org/foundation/mailinglists.html) for all official project communications.
```

Comment:
No need to duplicate this paragraph, as long as you link to the main mailing list page for Solr?